body.tex

```diff
@@ -232,6 +232,15 @@ \section*{Interoperability and extensibility}
 % Ralf: Could you provide some proof of effectiveness, e.g., "This has made it
 % possible to create distributed GPU arrays, enabling .... [find a reference to
 % work by Peter Entschev]".
+% Below an attempt to start on this:
+Thanks to these developments users of Dask and NumPy arrays are now capable
+to scale their computation to the GPU using Dasks CuPy integration with
+only minimal code changes.
```

Comment:
Honestly, I just realized I have to check that this tool chain is actually has `__array_function__` as a requirement... I assume this is the case though.

Reply:
Ah, OK, it is spelled out on this page: https://docs.dask.org/en/latest/gpu.html